R/ppmify.R

```diff
@@ -80,7 +81,9 @@ ppmify <- function (coords,
   # generate integration points
   int <- switch(method,
                 grid = grid(area, density),
-                count = NULL)
+                count = NULL,
+                bias = bias(area, bias_points),
+                biasRaster = bias(area, bias_points, offset.raster=TRUE))
```

Comment:
the rest of the code is using `snake_case` for arguments to functions. could you change `offset.raster` -> `offset_raster`?